src/cljs/rems/administration/create_catalogue_item.cljs

```diff
@@ -188,8 +188,22 @@
          :item-selected? item-selected?
          :on-change #(rf/dispatch [::set-selected-workflow %])}])]))
 
+(defn resource-label [r language counts]
+  (let [organisation (get-in r [:organization :organization/short-name language])
+        duplicate? (> (get counts (:resid r)) 1)
+        licenses? (not-empty (:licenses r))]
```

Comment:
The Clojure idiom for this is `(seq (:licenses r))`

Reply:
Thanks!